### PR TITLE
Add new section Monitoring

### PR DESCRIPTION
**Pull request by bouthilx**
### Add new section Monitoring
Why:

Monitoring of experiment is an important part of the usage of a tool for
hyper-parameter optimisation. There is many commands and ways to monitor
experiments using the library API, therefore it seams relevant to have a
section dedicated to this.

### Move EVC to bottom of menu 

Why:

EVC is arguably is less important than monitoring and algorithm configuration.

### CODE REVIEW

File: docs/src/user/cli/info.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
+``info`` Detailed informations about experiments
+------------------------------------------------
+
+This commands gives a detailed description of a given experiment.
```
Comment (DatCorno) [inline docs/src/user/cli/info.rst]: command*

File: docs/src/user/cli/info.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+   best trial:
+     /lr: 0.03543491957849911
+   best evaluation: 0.9626
+   start time: 2019-07-18 18:57:57.840000
+   finish time: 2019-07-18 18:58:08.565000
+   duration: 0:00:10.725000
+
+
+The last section contains information about the best trial so far, providing its
+hyper-parameter values and the corresponding objective
```
Comment (DatCorno) [inline docs/src/user/cli/info.rst]: hyperparameter*

File: docs/src/user/library/results.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+   # Database manually set
+   ExperimentBuilder().build_view_from(
+       {"name": "orion-tutorial",
+        "dataset": {
+            "type": "mongodb",
+            "name": "myother",
+            "host": "localhost"}})
+
+For a complete example, here's how you can fetch trials from a given experiment.
```
Comment (DatCorno) [inline docs/src/user/library/results.rst]: here is how

File: docs/src/user/library/results.rst [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+   # Fetches only the most recent trials using mongodb-like syntax
+   for trial in experiment.fetch_trials({'end_time': {'$gte': some_datetime}}):
+       print(trial.id)
+       print(trial.end_time)
+
+You can pass queries to :meth:`fetch_trials() <orion.core.worker.experiment.Experiment.fetch_trials>`,
+where queries can be a simple dictionary of values to
+match like ``{'status': 'completed'}``, in which case it would return all trials where
+``trial.status == 'completed'``, or they can be more complex using `mongodb-like syntax`_.
+You can find more in formation on the object :class:`Experiment <orion.core.worker.experiment.Experiment>` in the code
```
Comment (DatCorno) [inline docs/src/user/library/results.rst]: no space between in and formation.

File: docs/src/user/pytorch.rst [hunk 5]
```diff
@@ -225,65 +224,5 @@ following statistics if Oríon is called with ``-v`` or ``-vv``.
...
-       print(trial.end_time)
-
-You can pass queries to ``fetch_trials()``, where queries can be a simple dictionary of values to
-match like ``{'status': 'completed'}``, in which case it would return all trials where
-``trial.status == 'completed'``, or they can be more complex using `mongodb-like syntax`_.
-
-.. _`mongodb-like syntax`: https://docs.mongodb.com/manual/reference/method/db.collection.find/
-
-
+These results can be printed in terminal latter on with the command :ref:`info <cli-info>` or
```
Comment (DatCorno) [inline docs/src/user/pytorch.rst]: later*

File: docs/src/user/library/results.rst (hunk 4 above)
Comment (bouthilx) [reply docs/src/user/library/results.rst]: :rofl:

**Comment by bouthilx** [comment]
[Add monitoring section and migrate doc on result querying.](https://trello.com/c/GK28MheJ/201-add-monitoring-section-and-migrate-doc-on-result-querying)

**Comment by bouthilx** [comment]
[Document `orion info`](https://trello.com/c/yKNeUdQb/173-document-orion-info)

**Pull request merged**